README.md

```diff
@@ -220,3 +230,25 @@ devcontainer exec --workspace-folder . ./bin/test-local
 
 ![Showing Sublime Text on a Mac using the Dev Container CLI to run Crypteia tests.](/images/readme-devcontainer-cli-sublime.png)
 
+
+#### Troubleshooting
```

Comment:
This should go under `Usage` vs `Development`. The later is more about how to clone this project and contribute. Can you move it right under the `IAM Permissions` section?

Reply:
Sure